packages/server/middleware.js

```diff
@@ -1,8 +1,8 @@
+const bufflog = require('@bufferapp/bufflog');
+
 module.exports.apiError = (err, req, res, next) => {
   // eslint-disable-line no-unused-vars
-  console.log('-----');
-  console.error(err);
-  console.log('-----');
+  bufflog.error('Publish API error: '.err.message, err);
```

Comment:
I think there's a typo here, you probably want this:

```suggestion
  bufflog.error(`Publish API error: ${err.message}`, err);
```